src/create_objects.py

```diff
@@ -0,0 +1,62 @@
+import pandas as pd
```

Comment:
i am confused by this file, why are we plotting? 

we should be scraping in one function, then once we have scraped we can plot!

Reply:
bit of a misnomer with plot_util -- I created it for the map_plot, but it also contains the code for geocoding the addresses, hence my calling it for creating the Address objects. We can separate the geocoding into a different file, or rename this script if it's confusing.

Reply:
Here's what I'm going to do:
-rename "create_objects" this script "scrape.py"
functionality will be to scrape and geocode each address (throwing out addresses that fail to scrape or fail to geocode)
-move the geocode function into scrape_util to keep plotting separate.